.github/workflows/build.yml

```diff
@@ -23,11 +23,20 @@ jobs:
     steps:
     - uses: actions/checkout@v1
     
-    - name: dependencies (ubuntu and macos)
-      if: ${{ matrix.os == 'macos-latest' || matrix.os == 'ubuntu-latest' }}
+    - name: dependencies (macos)
+      if: ${{ matrix.os == 'macos-latest' }}
       run: |
+        brew uninstall openssl
+        rm -rf /usr/local/etc/openssl
+        rm -rf /usr/local/etc/openssl@1.1
```

Comment:
Are these lines really needed?

Reply:
i added to ensure it goes clean .. but may be its fine and they get overwritten